oxd-server/src/main/java/org/gluu/oxd/server/Cli.java

```diff
@@ -293,4 +299,35 @@ private static Options options() {
 
         return options;
     }
+
+    private static boolean checkForMultipleOptions(CommandLine cmd){
```

Comment:
when we say `check*` in method name, then usually it's expected to throw exception. I propose to rename it to `hasMultipleActionOptions`.

Reply:
checkForMultipleOptions is renamed to hasMultipleActionOptions